microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/ValidationType.java

```diff
@@ -24,7 +24,7 @@
  */
 public enum ValidationType {
 
-	syntax, unknown, duplicate, value, required, requiredValue;
+	syntax, unknown, duplicate, value, required, requiredValue, missingCloseBrace, unknownReference;

```

Comment:
The validation type is bound with settings `microprofile.tools.validation.*` https://github.com/redhat-developer/vscode-microprofile#supported-vs-code-settings. I think we should:

 * remove missingCloseBrace and replace it with syntax.
 * rename `unknownReference` with `expression` and after that we should expose a new settings in vscode-microprofile.

Reply:
Opened https://github.com/redhat-developer/vscode-microprofile/pull/18 for the client changes for the new setting.